src/engine/GameConstants.java

```diff
@@ -35,21 +35,28 @@
      * 13/2/2017
      */
     public static final float SHIP_SMALL_WIDTH = 41;
-    public static final float SHIP_SMALL_LENGTH =71;
+    public static final float SHIP_SMALL_LENGTH = 71;
     public static final float SHIP_MEDIUM_WIDTH = 61;
     public static final float SHIP_MEDIUM_LENGTH = 102;
     public static final float SHIP_LARGE_WIDTH = 76;
     public static final float SHIP_LARGE_LENGTH = 129;
     public static final float SHIP_BASE_RAILS = 2;
     public static final float SQUADRON_PLASTIC_RADIUS = 12.7f;
-    public static final float SQUADRON_CARDBOARD_RADIUS =12.4f;
+    public static final float SQUADRON_CARDBOARD_RADIUS = 12.4f;
 
 
-    /**Layers, how far from the camera the associated items are drawn
-     *
+    /**
+     * Layers, how far from the camera the associated items are drawn
      */
     public static final int SQUADRON_PLASTIC = 25;
     public static final int SQUADRON_CARDBOARD = 24;
     public static final int SQUADRON_GRAPHIC = 22;
     public static final int MAP_BACKGROUND_LAYER = 20000;
-}
+
+    /**
+     *Defense Token Constants
+     */
+    public enum defenseTokenType {BRACE, REDIRECT, EVADE, SCATTER, CONTAIN, SALVO}
+    public enum defenseTokenStatus {READY, EXHAUSTED, DISCARDED}
```

Comment:
I'm not wild about these living here. Mostly because this is a home for "magic numbers" so that we can change things on the fly.  Worth mentioning that I didn't know you could do this.  I thought they needed their own file.

I do know that in the AttackPool PR I made at least one other enum without added functionality to go with it.  It might be worth making an issue to create a file for boring enums like this to live next to the GameConstants once this and that are merged.

Reply:
Don't worry about changing this.

Reply:
heard